File: src-scala-orig/test/scala/cern/colt/matrix/tint/impl/SparseRCIntMatrix2DViewTest.scala
package cern.colt.matrix.tint.impl

//remove if not needed
import scala.collection.JavaConversions._

class SparseRCIntMatrix2DViewTest(arg0: String) extends SparseRCIntMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new SparseRCIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new SparseRCIntMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new SparseRCIntMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tint/impl/SparseRCIntMatrix2DViewTest.java
package cernx.colt.matrix.tint.impl;

public class SparseRCIntMatrix2DViewTest extends SparseRCIntMatrix2DTest {

    public SparseRCIntMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseRCIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseRCIntMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseRCIntMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
